en/docs/learn/api-controller/ci-cd-with-wso2-api-management.md

```diff
@@ -207,16 +207,101 @@ For more information on initializing an API Project using OpenAPI/Swagger Specif
 The Automation Server can be configured to run a specific pipeline for promoting artifacts to other environments. 
 The DevOps team can develop this pipeline further to include automated tests, workflow approvals, and other tasks.  
 
-The **apictl** tool should be installed in the automation servers to begin the process. As the tool supports a variety of platforms, including Linux/Windows and macOS, this can be done easily. The **apictl** tool supports the process of importing API Projects via the `import-api` command. This command is mainly used in the pipeline to migrate to different environments.
+The **apictl** tool should be installed in the automation servers to begin the process. As the tool supports a variety of platforms, including Linux/Windows and macOS, this can be done easily. 
 
-1.  Import the `SwaggerPetstore` API into the production environment and test the API by running the following sample command.
+#### a. Promoting API project(s) in a GIT repository using a unified approach
+
+The repository (Version Control System) needs to be cloned into the instance that is executing the CI/CD process. From 3.2.0, **apictl** has the inbuilt support to integrate with a GIT based version control system. It gives a unified command `vcs deploy` to deploy any type of project (APIs, API Products and Apps).
+
+1.  Change your working directory (`cd`) into the cloned git repository.
```

Comment:
Should this be like below or I am not clear this sentence :)
```suggestion
1.  Change your working directory and (`cd`) into the cloned git repository.
```

Reply:
Its, `cd <repo-name>` :)

Reply:
ahh :), shall we add that part too to sth like this?
Change your working directory and navigate into the cloned git repository (cd <repo-name>)